src/main/java/seedu/address/model/person/Doctor.java

```diff
@@ -9,24 +10,74 @@
  */
 public class Doctor extends Person {
 
+    private final ShiftDays shiftDays;
+
     /**
      * Every field must be present and not null.
      */
     public Doctor(Name name, Phone phone, Email email,
         Set<Affiliation> affiliations, Set<Affiliation> affiliationHistory) {
         super(name, phone, email, new Role("Doctor"), affiliations, affiliationHistory);
+
+        // Instantiation of Doctor should always begin with empty shift days.
+        shiftDays = new ShiftDays();
     }
 
     /**
      * Every field must be present and not null, except affiliationHistory.
      */
     public Doctor(Name name, Phone phone, Email email, Set<Affiliation> affiliations) {
         super(name, phone, email, new Role("Doctor"), affiliations);
+
+        // Instantiation of Doctor should always begin with empty shift days.
+        shiftDays = new ShiftDays();
```

Comment:
Important to take note of when implementing further shift changes, thanks for noting in a comment!